### PR TITLE
Main page logo changed by screenshot

### PR DESCRIPTION
**Pull request by Nivedita967**
I have replaced Logo in the main page by screenshot. Check if it is looking better or not.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -143,8 +143,11 @@ <h1>
               <div class="image-container">
                 <img
                   class="img-fluid"
-                  src="img/ic_launcher-web.png"
+                  src="img/shot_1.webp"
```
Comment (velitasali) [inline index.html:146]: It might look nicer if the screenshot were showing an in-app view IMO. Other than that, it is approvable.

Comment (ptanmay143) [reply index.html:146]: I agree, a screenshot of maybe the home page when the app is opened would be preferred over the web UI of trebleshot. [This](https://trebleshot.monora.org/img/shot3.webp) screenshot seems to be a better candidate.

**Comment by ptanmay143** [comment]
- It would be better if the screenshot could have been wrapped inside a phone vector. See [this](https://placeit.net/c/mockups/stages/iphone-x-ios-screenshot-generator-1321) example.
- The GitHub icon in the top right (and the one at the extreme bottom) has almost zero visibility due to the background and foreground colors cancelling each other out.
- The stats at the bottom of the page have been changed as follows:
Issues: 68 -> 79
Stars: 417 -> 570
Downloads: Play Store still shows 10,000+ downloads. 👍

File: index.html (hunk 1 above)
Comment (Nivedita967) [reply index.html:146]: Okay, I am trying to do changes according to you.

**Comment by Nivedita967** [comment]
I have done changes, please review.

**Review by velitasali — APPROVED** [review]
It looks good. Thanks for the help. 

BTW, there is an unnecessary file named "git" in the root folder, but it is not a dealbreaker :)

**Comment by velitasali** [comment]
> * The GitHub icon in the top right (and the one at the extreme bottom) has almost zero visibility due to the background and foreground colors cancelling each other out.

We can discuss this by opening another issue.

> * The stats at the bottom of the page have been changed as follows:
>   Issues: 68 -> 79
>   Stars: 417 -> 570
>   Downloads: Play Store still shows 10,000+ downloads. +1

Yes, I mentioned it in #7.

**Comment by Nivedita967** [comment]
> It looks good. Thanks for the help.
> 
> BTW, there is an unnecessary file named "git" in the root folder, but it is not a dealbreaker :)

I have not added this git file and don't even noticed. You may delete this :)

**Pull request merged**